client/packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -175,7 +177,6 @@ const DataTableComponent = <T extends RecordWithId>({
             {(row, idx) => (
               <DataRow
                 key={row.id}
-                rows={data}
```

Comment:
So you've removed `rows` from everywhere in the DataTable -- is that because nothing was actually using it?

Presumably this was included originally so individual rows would have access to other rows data, and you can't see a use case for it currently?

Reply:
exactly right - it was used but I can't even remember what for. couldn't see anything that was using it now though. and it's a bit dodgy anyway - I think that rows should only be working with their own data.